.github/workflows/ci.yml

```diff
@@ -148,7 +148,7 @@ jobs:
         uses: actions-rs/cargo@v1
         with:
           command: bench
-          args: -F bench
+          args: -F bench --no-run
```

Comment:
why are we disabling the benchmark running? I guess we just want to make sure they compile, but not actually measure in the CI?

Reply:
Right... just to try to improve the overall running time of the CI (so that it doesn't take forever)

Reply:
.. that doesn't seem right lol. 13min sounds like it's running the full benchmarks not just compiling the bench's. Let's see how the CI does on my pr